scripts/build-android.sh

```
@@ -0,0 +1,238 @@
+            # ImageMagick: resize PNG then convert to WEBP
+            convert "$ICON_SOURCE" -resize "${size}x${size}" "$dir/ic_launcher.webp"
+            convert "$ICON_SOURCE" -resize "${size}x${size}" "$dir/ic_launcher_round.webp"
+            # Foreground layer (with padding for adaptive icon safe zone)
+            local fg_size=$((size * 108 / 72))
+            convert "$ICON_SOURCE" -resize "${size}x${size}" \
+                -gravity center -background none -extent "${fg_size}x${fg_size}" \
+                "$dir/ic_launcher_foreground.webp" 2>/dev/null
+        elif [ "$tool" = "cwebp" ]; then
+            # cwebp: need intermediate PNG via ffmpeg or other
+            local tmp_png="$tmp_dir/ic_launcher_${density}.png"
+            local tmp_fg_png="$tmp_dir/ic_launcher_fg_${density}.png"
```

Shellcheck (0.11.0)

[warning] tmp_fg_png appears unused. Verify use (or export if used externally).

(SC2034)